CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.5.0 (2017-11-03)
+
+- Added `SkyAppOmnibarProvider` to enable custom `envId` and `svcId` to be provided to the Omnibar. [#323](https://github.com/blackbaud/skyux-builder/pull/323)
+- Cleaned up logging by only generating config for known commands. [#321](https://github.com/blackbaud/skyux-builder/pull/321)
+- Enabling default `envid` and `svcid` params via `skyuxconfig.json`. [#320](https://github.com/blackbaud/skyux-builder/pull/320)
+- Exposing `--platform` flag and adjusting config paths to facilitate better internal Continuous Integration. [#297](https://github.com/blackbaud/skyux-builder/pull/297), [#322](https://github.com/blackbaud/skyux-builder/pull/322)
```

Comment:
I'd change "Exposing" to "Exposed" and "adjusting" to "adjusted" to consistently use past tense.

I'd change "facilitate better" to "improve."

I think "continuous integration" should be lowercase.